pytensor/tensor/rewriting/elemwise.py

```diff
@@ -380,6 +380,99 @@ def is_dimshuffle_useless(new_order, input):
     return is_useless
 
 
+@node_rewriter([Elemwise])
+def local_elemwise_lift_scalars(fgraph, node):
+    op = node.op
+
+    if not isinstance(op, Elemwise):
```

Comment:
We don't need the isisntance checks anymore. PyTensor only calls the rewrite if the node is an instance of "tracks" in the decorator

Reply:
fixed